release.config.js

```diff
@@ -20,7 +20,7 @@ module.exports = {
     [
       '@semantic-release/git',
       {
-        assets: ['CHANGELOG.md', 'package.json', 'package-lock.json'],
+        assets: ['CHANGELOG.md', 'docs/', 'package.json', 'package-lock.json'],
```

Comment:
publishing all content in the docs folder in case we add other documentation later.